src/internal/innies-and-outies.ts

```diff
@@ -0,0 +1,114 @@
+/** @prettier */
```

Comment:
IMO, this file should be named something else, as "innies and outies" isn't a comparison that's made only between belly buttons and this has the potential to make some people uncomfortable.

Reply:
I'm not sure I want to be friends with people who are uncomfortable about bellybuttons.

But I'll change it.